streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/IngressController.scala

```diff
@@ -93,16 +115,6 @@ object IngressController extends Logger {
     }
   }
 
-  def deleteIngress(ingressName: String, nameSpace: String): Unit = {
```

Comment:
Perhaps we should keep the independent deletion method for Ingress and allowing users to manually delete or update Ingress behaviors as needed?